bot.py

```diff
@@ -756,6 +780,49 @@ def trim_content_to_length(text, link, max_length=1000):
         with open(NewsDownloader.NEWS_FILENAME, 'w') as f:
             f.write('[]')
 
+    async def change_language(self, message, new_language, prefix, lang):
+        my_language = self.language.get(message.guild)
+        if not message.guild:
+            e = discord.Embed(title='Language change', color=self.RED)
+            e.add_field(name='Error',
+                        value=f'Language change not possible in direct messages.')
+            await answer(message, e)
+            return
+        if self.is_guild_admin(message):
+            if len(new_language) != 2:
+                e = discord.Embed(title='Language change', color=self.RED)
+                e.add_field(name='Error',
+                            value=f'Your new language has to be 2 characters long, `{new_language}` has {len(new_language)}.')
+                await answer(message, e)
+                return
+            if new_language not in Translations.LANGUAGES:
+                e = discord.Embed(title='Language change', color=self.RED)
+                e.add_field(name='Error',
+                            value=f'Your new language is not available, `{new_language}`.')
+                await answer(message, e)
+                return
+
+            self.language.add(message.guild, new_language)
+            e = discord.Embed(title='Administrative action', color=self.RED)
+            e.add_field(name='Language change', value=f'Language was changed from `{my_language}` to `{new_language}`')
```

Comment:
The scope of the change is not clear. Let's make it easy to understand that it's default for the whole server now.